docs/speech-usage.rst

```diff
@@ -196,10 +196,10 @@ See: `Single Utterance`_
     ...                            sample_rate=16000)
     ...     responses = client.streaming_recognize(sample,
     ...                                            single_utterance=True)
-    ...     alternatives = list(responses)
-    >>> print(alternatives[0].transcript)
+    ...     results = list(responses)
+    >>> print(results[0].alternatives[0].transcript)
     hello
-    >>> print(alternatives[0].confidence)
+    >>> print(results[0].alternatives[0].confidence)
```

Comment:
I think you meant "would" instead of "wouldn't" so I'll make a python snippet.

```python
#  Previously 
for alternatives in client.streaming_recognize(sample, interim_results=True):
    print('=' * 20)
    print(alternatives[0].transcript)
    print(alternatives[0].confidence)
    ====================
    'hello'
    0.973458576
```

```python
# Now
for results in client.streaming_recognize(sample, interim_results=True):
    print('=' * 20)
    print(results[0].alternatives[0].transcript)
    print(results[0].alternatives[0].confidence)
    print(results[0].is_final)
    print(results[0].stability)
====================
'hello'
0.973458576
True
0.982345
```

Since the `is_final` and `stability` values don't make sense directly in the `alternative` data, they *should* go one level up AFAICT.